corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -34,18 +34,22 @@ <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
 {% endblock %}
 
 {% block page_content %}
-    <table id="data-dictionary-table" class="table">
-        <thead>
-            <th>Case Property</th>
-            <th>Type</th>
-            <th>Description</th>
-        </thead>
-        <tbody data-bind="foreach: activeCaseTypeData">
-            <tr>
-                <td data-bind="text: name"></td>
-                <td data-bind="text: type"></td>
-                <td data-bind="text: description"></td>
-            </tr>
-        </tbody>
-    </table>
+    <div class="row">
+        <div class="col-md-10">
+            <table id="data-dictionary-table" class="table">
+                <thead>
+                    <th>{% trans "Case Property" %}</th>
+                    <th>{% trans "Type" %}</th>
```

Comment:
Cool. That makes sense then. I'd maybe update that to say "Data Type" whenever you build this out since "Type" is an overloaded word when we are thinking about cases. But that doesn't have to happen now.